include/ap/argument_parser.hpp

```diff
@@ -179,6 +175,10 @@ class optional_argument : public argument_interface<T> {
         return this->_name == other._name;
     }
 
+    [[nodiscard]] const std::optional<std::string_view> short_name() const {
```

Comment:
Move this function below the `name()` function
Add override to the `name()` function